src/http/structs.rs

```diff
@@ -100,9 +100,24 @@ pub struct EntityPath {
     pub entity: String,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityProfileLink {
+    pub url: String,
+    pub verified: bool,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityProfile {
+    pub display_name: Option<String>,
+    pub description: Option<String>,
+    pub workplace: Option<String>,
```

Comment:
drastically overthinking it: do we want to more generically call it an `organization`, in case someone has affinity to it but doesn't identify as working there?

Reply:
That's the word I was looking for! I'll change it :)